src/App.tsx

```diff
@@ -17,18 +46,20 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter query={query} filterBy={filterBy} setFilterBy={setFilterBy} setQuery={setQuery} applyQuery={applyQuery} />
```

Comment:
don't pass function for updating state in child component (setFilterBy, setQuery), create a functions wrapper for them